versions.props

```diff
@@ -5,8 +5,8 @@ org.apache.hadoop:* = 2.7.3
 org.apache.hive:hive-metastore = 2.3.6
 org.apache.orc:* = 1.6.2
 org.apache.parquet:parquet-avro = 1.11.0
-org.apache.spark:spark-hive_2.11 = 2.4.4
-org.apache.spark:spark-avro_2.11 = 2.4.4
```

Comment:
Should this be `org.apache.spark:* = 2.4.5`? I'm not sure why we are pinning just the two modules.